sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/Schema.java

```diff
@@ -410,14 +410,13 @@ public boolean isSupertypeOf(TypeName other) {
     // Returns the type of this field.
     public abstract TypeName getTypeName();
 
+    // Whether this type is nullable.
+    public abstract Boolean getNullable();
```

Comment:
I wonder if a convenience method `FieldType nullable() { return toBuilder().setNullable(true).build(); }` would make sense

Reply:
@akedin in fact there is such a method, but it's called withNullable